operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -174,8 +174,6 @@ type CassandraDatacenterSpec struct {
 
 	AdditionalSeeds []string `json:"additionalSeeds,omitempty"`
 
-	Reaper *ReaperConfig `json:"reaper,omitempty"`
```

Comment:
what happens if I had this and I was using 1.5.0, then I upgrade and we rip it out?

Reply:
That's a good question. I will test and report back my findings.